src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionAboutToSubmitHandler.java

```diff
@@ -112,6 +105,22 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         return preSubmitCallbackResponse;
     }
 
+    private boolean anyHearingsScheduledInTheFuture(SscsCaseData caseData) {
```

Comment:
might be better to rename to "hasHearingScheduledInTheFuture" or "isHearingScheduledInTheFuture"